2021-01-11 16:12:40,849 INFO    MainThread:10916 [wandb_setup.py:_flush():70] setting env: {}
2021-01-11 16:12:40,849 INFO    MainThread:10916 [wandb_setup.py:_flush():70] multiprocessing start_methods=spawn
2021-01-11 16:12:40,849 INFO    MainThread:10916 [wandb_setup.py:_flush():70] setting login settings: {}
2021-01-11 16:12:40,849 INFO    MainThread:10916 [wandb_init.py:_log_setup():303] Logging user logs to C:\Users\kevin\OneDrive\Documents\8th sem\Federated_Pneumonia\src\client\wandb\run-20210111_161240-3cmt5yng\logs\debug.log
2021-01-11 16:12:40,849 INFO    MainThread:10916 [wandb_init.py:_log_setup():304] Logging internal logs to C:\Users\kevin\OneDrive\Documents\8th sem\Federated_Pneumonia\src\client\wandb\run-20210111_161240-3cmt5yng\logs\debug-internal.log
2021-01-11 16:12:43,215 INFO    MainThread:10916 [wandb_run.py:_console_start():1237] atexit reg
2021-01-11 16:12:43,216 INFO    MainThread:10916 [wandb_run.py:_redirect():1104] redirect: SettingsConsole.WRAP
2021-01-11 16:12:43,216 INFO    MainThread:10916 [wandb_run.py:_redirect():1135] Wrapping output streams.
2021-01-11 16:12:43,216 INFO    MainThread:10916 [wandb_run.py:_redirect():1151] Redirects installed.
2021-01-11 16:12:56,909 INFO    MainThread:10916 [wandb_run.py:_atexit_cleanup():1207] got exitcode: 1
2021-01-11 16:12:56,909 INFO    MainThread:10916 [wandb_run.py:_restore():1179] restore
2021-01-11 16:12:56,909 INFO    MainThread:10916 [wandb_run.py:_wait_for_finish():1324] got exit ret: uuid: "1ecb423e8af54d87871208e86b7a1647"
response {
  poll_exit_response {
    file_counts {
      wandb_count: 1
    }
    pusher_stats {
      uploaded_bytes: 938
      total_bytes: 938
    }
  }
}

2021-01-11 16:12:59,022 INFO    MainThread:10916 [wandb_run.py:_wait_for_finish():1324] got exit ret: uuid: "aadc18ea4223488385b8c7f6f0ae6d35"
response {
  poll_exit_response {
    file_counts {
      wandb_count: 5
    }
    pusher_stats {
      uploaded_bytes: 6438
      total_bytes: 6438
    }
  }
}

2021-01-11 16:13:01,053 INFO    MainThread:10916 [wandb_run.py:_wait_for_finish():1324] got exit ret: uuid: "27eb56883a144e24b7e45dfca673116f"
response {
  poll_exit_response {
    done: true
    exit_result {
    }
    file_counts {
      wandb_count: 5
    }
    pusher_stats {
      uploaded_bytes: 6438
      total_bytes: 6438
    }
  }
}

2021-01-11 16:13:02,209 INFO    MainThread:10916 [wandb_run.py:_show_summary():1469] rendering summary
2021-01-11 16:13:02,209 INFO    MainThread:10916 [wandb_run.py:_show_history():1507] rendering history
2021-01-11 16:13:02,209 INFO    MainThread:10916 [wandb_run.py:_show_files():1536] logging synced files
2021-01-11 16:13:02,209 INFO    MainThread:10916 [internal.py:handle_exit():138] Internal process exited
